clupload/cluploadArduino101_linux.sh

```diff
@@ -24,17 +24,17 @@ echo "BIN FILE" $bin_file_name
 DFU=$fixed_path/dfu-util
 echo "wating for Arduino 101 device... "
 COUNTER=0
-f=`$DFU -l -d 8087:0ABA | grep sensor_core | cut -f 1 -d ' '`
+f=`$DFU -l -d ,8087:0ABA | grep sensor_core | cut -f 1 -d ' '`
```

Comment:
I don't think it matters. I've seen it implemented both ways. The comma separates the PID/VID and the DFU PID/VID:
 -d --device <vendor>:<product>[,<vendor_dfu>:<product_dfu>]